src/orchestrator/graph.py

```
@@ -83,6 +89,45 @@ async def orchestrate(state: State, config: RunnableConfig):
+    Returns:
+        An asynchronous function that processes the task_manager and returns a dictionary with a tool message containing the changes.
+    """
+
+    async def task_manager(state: State, config: RunnableConfig, store: BaseStore):
+        tool_call = state.messages[-1].tool_calls[0]
+        config_with_recursion = RunnableConfig(**config)
+        config_with_recursion["recursion_limit"] = recursion_limit
+
+        result = await task_manager_graph.compiled_graph.ainvoke(
+            TaskManagerState(
+                messages=[HumanMessage(content=tool_call["args"]["content"])]
```

Ruff (0.11.9)

Undefined name `HumanMessage`

(F821)